### DISCUSSION TITLE
when i try to use pdf.to_image() ,get an error like this

### DISCUSSION THREAD

**Discussion by fangjiyuan** [jsvine/pdfplumber#949] [Q&A]
when i try to use pdf.to_image() ,get an error like this:  
--------
 **** Error: Form stream has unbalanced q/Q operators (too many q's)
               Output may be incorrect.
--------
version:
pdfplumber 0.9.0
ghostscript-9.22
python 3.7.3
ImageMagick 7.1.1-5

**Comment by cmdlineluser** [comment]
Hi @fangjiyuan 

pdfplumber has recently updated `.to_image()` - perhaps you could try the latest version `0.10.1` to see if the issue is still there?

Or, if possible - could you provide the PDF file that causes the error?

**Reply by fangjiyuan** [reply]
thanks for your reply. I will check out if  the latest version 0.10.1 can make it .
